acquia_lift.module

```diff
@@ -109,6 +109,7 @@ function acquia_lift_page_build(&$page) {
     _acquia_lift_add_meta_tag('authEndpoint', variable_get('acquia_lift_oauth_url'));
   }
   _acquia_lift_add_meta_tag('contentReplacementMode', variable_get('acquia_lift_content_replacement_mode', 'untrusted'));
+  _acquia_lift_add_meta_tag('bootstrapMode', variable_get('acquia_lift_bootstrap_mode', ''));
```

Comment:
Please rotate line 111 and 112 to keep the same order between bootstrap_mode and content_replacement_mode across the rest of the applications.

Reply:
Should we default this to 'auto' here?

Reply:
yes, we need to make sure that behavior is the same as current by default. Think of it as opt out.